src/rating.css

```diff
@@ -0,0 +1,33 @@
+.card__rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 16px 24px;
+}
+
+.stars {
+  display: flex;
+}
```

Comment:
let's leave the styles for `stars` only (and rename the file to `starts.css` please).
the `.card__rating` selector belongs to card block so move it to `card.css`